src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -335,6 +388,11 @@ private void create()
     newSmartContract = newSmartContract.toBuilder()
         .setContractAddress(ByteString.copyFrom(contractAddress)).build();
     long callValue = newSmartContract.getCallValue();
+
+    if (isNewVersion() && callValue < 0) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Change this condition so that it does not always evaluate to "false" ([squid:S2583](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2583))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbSdHUuSgN4uCclEWnC&open=AWbSdHUuSgN4uCclEWnC)

[ISSUE_KEY]: AWbSdHUuSgN4uCclEWnC